src/interfaces/HposInterface.js

```diff
@@ -91,7 +93,11 @@ const presentHposSettings = (hposSettings) => {
 const HposInterface = {
   hostedHapps: async () => {
     const result = await hposHolochainCall({ method: 'get', path: '/hosted_happs' })()
-    return result.hosted_happs.map(mergeMockHappData)
+    if (Array.isArray(result)) {
+      return result.map(mergeMockHappData)
+    } else {
+      return []
+    }
```

Comment:
This logic seems weird. What does it mean if the value is not an array? Should we be logging an error?